### PR TITLE
Persistent navbar

### PR DESCRIPTION
**Pull request by kevinylee1991**
Looking for suggestions regarding the showNavMenu and showNavBar stuff (left commented code in the review for easy reference) -- are these still needed? Which views with showNavBar be false for, maybe login?

### CODE REVIEW

File: src/app/core/common/Navbar.js [hunk 1]
```diff
@@ -98,22 +132,20 @@ class Navbar extends React.Component {
...
-            <IconButton onClick={this.handleDrawerClose}>
-              <ChevronLeftIcon />
-            </IconButton>
-          </div>
-          <Divider />
-          <MenuList>
-            {links.map(this.renderNavLink)}
-          </MenuList>
+        <div className={classes.drawerHeader}>
+          <IconButton onClick={this.handleDrawerClose}>
```
Comment (HF-liu) [inline src/app/core/common/Navbar.js]: Maybe we could use one function to replace both `handleDrawerClose` and `handleDrawerOpen`? Like:

```
handleDrawerStatus = () => {
    this.setState(prevState => ({
      open: !prevState.open
    }))
  }
```

File: src/app/App.js [hunk 2]
```diff
@@ -25,33 +25,57 @@ class App extends React.Component {
...
+    //         </div>
+    //         {showFooter && renderFooter()}
+    //       </div>
+    //     </MuiThemeProvider>
+    //   </Router>
+    // )
+
     return (
       <Router>
         <MuiThemeProvider theme={theme}>
```
Comment (HF-liu) [inline src/app/App.js]: Picked theme color from Platform9.com:

```
const theme = createMuiTheme({
      palette: {
        type: this.props.theme,
        primary: {
          light: '#66CCF1',
          main: '#00ABE8',
          dark: '#007AB6',
          contrastText: '#fff',
        },
      }
    })
```

File: src/app/App.js [hunk 3]
```diff
@@ -25,33 +25,57 @@ class App extends React.Component {
...
           <div id="_main-container">
-            {showNavMenu && renderNavMenu()}
-            {showSidebar && renderSidebar()}
-            <div id="_main-content">
+            <Navbar links={pluginManager.getNavItems()} >
```
Comment (brennancheung) [inline src/app/App.js]: weird space at end, probably going to trip up linting

**Comment by kevinylee1991** [comment]
I tweaked the color hashes just a bit using the ui styleguide as a reference

**Pull request merged**